configs/yandex_clickhouse.json

```diff
@@ -20,16 +20,29 @@
       "selector": "/html/@lang",
       "type": "xpath",
       "global": true
+    },
+    "version": {
+      "selector": "/html/@data-version",
+      "type": "xpath",
+      "global": true
+    },
+    "single-page": {
+      "selector": "/html/@data-single-page",
+      "type": "xpath",
+      "global": true
     }
   },
   "custom_settings": {
     "attributesForFaceting": [
-      "lang"
+      "lang",
+      "version",
+      "single-page"
     ]
   },
   "min_indexed_level": 2,
   "conversation_id": [
     "1058861561"
   ],
-  "nb_hits": 50105
-}
\ No newline at end of file
+  "nb_hits": 50105,
+  "ignoreRobotsTxtRules": true
```

Comment:
What is the use of this parameter? It is not supported from our scraper IIRC

Reply:
@s-pace I've found it over here: https://www.algolia.com/doc/api-reference/crawler/configuration/ignore-robots-txt-rules/

Reply:
Thanks for the details. The URL you are referring to is the enterprise crawler offered by Algolia. This is another tool completely different from the DocSearch Crawler. [You can find out more information on this page](https://www.algolia.com/products/crawler/).

The documentation related to the DocSearch crawler is available [from this dedicated documentation](https://community.algolia.com/docsearch/config-file.html). The DocSearch crawler does not rely on the robots.txt so no worries.

Reply:
Please remove this attribute from the configuration so that I can deploy.

Reply:
@s-pace done, thanks